### PR TITLE
Add venv instructions to README

### PR DESCRIPTION
**Pull request by matthew55**
No idea if this is necessary, or even accurate but I am tired of seeing this needed explained in the Discord.

Let me know if anything needs changed. I have no idea because I use an IDE that automatically sets this up for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,16 +58,27 @@ Docker Note: If you make any changes to your `.env` file, you will need to resta
 
 ### Discord Bot: Python Script 🏃‍♀️
 1. Install Python 3.10 or higher
-2. Install the required packages:
+2. Setup a python virtual environment:
+#### Linux & Mac
+```bash
+python -m venv autorsa
+source ./autorsa/Scripts/activate
```
Comment (PZES) [inline README.md:65]: its should be ./autorsa/bin/activate

Comment (matthew55) [reply README.md:65]: image

It must be sourced as seen when looking at the script's code.

**Comment by PZES** [comment]
change the venv dir to something that ends with venv or add it to the gitignore

File: README.md (hunk 1 above)
Comment (PZES) [reply README.md:65]: i meant that the path is bin not scripts

**Comment by matthew55** [comment]
> change the venv dir to something that ends with venv or add it to the gitignore

Good catch!

File: README.md [hunk 2]
```diff
@@ -58,16 +58,27 @@ Docker Note: If you make any changes to your `.env` file, you will need to resta
 
 ### Discord Bot: Python Script 🏃‍♀️
 1. Install Python 3.10 or higher
-2. Install the required packages:
+2. Setup a python virtual environment:
+#### Linux & Mac
+```bash
+python -m venv autorsa-venv
+source ./autorsa-venv/Scripts/activate
```
Comment (PZES) [inline README.md:65]: The path is bin not scripts

**Review by PZES — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (NelsonDane) [reply README.md:65]: I'm updating this rn

**Comment by NelsonDane** [comment]
I thought it fit better under pre-setup, but now I'm thinking that could be wrong since if they wanted a docker only approach then they shouldn't need that

**Comment by matthew55** [comment]
> I thought it fit better under pre-setup, but now I'm thinking that could be wrong since if they wanted a docker only approach then they shouldn't need that

Yeah, it should not be necessary for anyone using docker. I think where it was is fine.

**Comment by NelsonDane** [comment]
> > I thought it fit better under pre-setup, but now I'm thinking that could be wrong since if they wanted a docker only approach then they shouldn't need that
> 
> Yeah, it should not be necessary for anyone using docker. I think where it was is fine.

Fixed now, I jumped the gun. My bad

**Comment by NelsonDane** [comment]
Is this good or do you have other suggestions @matthew55 @PZES

**Comment by matthew55** [comment]
Looks good to me

**Comment by PZES** [comment]
looks good

**Review by NelsonDane — APPROVED** [review]

**Pull request merged**